tex/rep-theory/semisimple.tex

```diff
@@ -228,8 +317,11 @@ \section{Semisimple algebras}
 	\end{enumerate}
 \end{theorem}
 \begin{proof}
-	(i) $\implies$ (ii) follows
-	from \Cref{thm:rep_1mat} and \Cref{prop:rep_direct_sum}.
+	(i) $\implies$ (ii) follows from
+	using \Cref{prop:rep_direct_sum} to breaks any finite-dimensional
+	representation of $A$ into a direct sum of representations of
+	$\Mat_{d_i}(k)$, then \Cref{thm:rep_1mat} shows any such representations are
+	completely reducible.
 	(ii) $\implies$ (iii) is tautological.
```

Comment:
Might as well mention here "because Reg(A) is finite-dimensional", but letting the reader unroll definitions is also okay.